addons/website/static/tests/tours/edit_menus.js

```diff
@@ -174,6 +176,17 @@ wTourUtils.registerWebsitePreviewTour('edit_menus', {
         trigger: '.oe_menu_editor li:contains("Home") ul li:contains("Contact us")',
         run: () => {}, // It's a check.
     },
+    // Drag the Mega menu to the first position.
+    {
+        content: "Drag Mega at the top",
+        trigger: '.oe_menu_editor li:contains("Megaaaaa!") > .ui-sortable-handle',
+        run: "drag_move_and_drop [0,0]@.oe_menu_editor li:contains('Home') > .ui-sortable-handle => .oe_menu_editor li",
+    },
+    {
+        content: "Wait for drop",
+        trigger: '.oe_menu_editor:first-child:contains("Megaaaaa!")',
+        run: () => {}, // It's a check.
+    },
```

Comment:
@loco-odoo I had to add this in order to prevent the mega menu to be in the + menu